### DISCUSSION TITLE
Document jira in Dojo more info for details..

### DISCUSSION THREAD

**Discussion by undelete** [DefectDojo/django-DefectDojo#4589] [Q&A] [answered]
Implementation issues for jira and dojo
Some events are not displayed in the document and cause problems in the inputs. Because jira expects certain courses and dojo the same and there is no exact documentation of the courses 

The solution is to add more details for integrations for dojo and jira in which specific element they should sync. if you have more examples of jira and dojo integrations, this will help more people.

**Comment by damiencarol** [comment]
@valentijnscholten know more the JIRA feature.

**Comment by undelete** [comment]
Great, @valentijnscholten , where find the information?

**Comment by valentijnscholten** [comment, marked as answer]
There's now a lot of documentation: https://docs.defectdojo.com/en/share_your_findings/jira_integration/connect_to_jira/
